ColorPatterns/Scenes/Game/GameSounds.swift

```diff
@@ -12,13 +12,15 @@ struct GameSounds {
     
     private var audioPlayer: AVAudioPlayer?
 
-    private let soundsArray = [K.Sounds.note1, K.Sounds.note2, K.Sounds.note3, K.Sounds.note4, K.Sounds.note5]
+    private let soundsArray = [Constants.Sounds.note1, Constants.Sounds.note2, Constants.Sounds.note3, Constants.Sounds.note4, Constants.Sounds.note5]
     
     mutating func play(soundFile: Int) {
         guard let soundURL = Bundle.main.url(forResource: soundsArray[soundFile],
-                                             withExtension: K.Sounds.wavFormat) else { return }
+                                             withExtension: Constants.Sounds.wavFormat) else { return }
         do {
             try audioPlayer = AVAudioPlayer(contentsOf: soundURL)
+            // TODO: You're creating new audio player for each sound? why?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/c07cd9a3b7a63e61e86332aeb4fa4cabf8b81c17/ColorPatterns/Scenes/Game/GameSounds.swift#L22"></span>TODOs should be resolved (You’re creating new audio play…).<br />
<code>todo</code> <code>GameSounds.swift:22</code></td>
    </tr>
  </tbody>
</table>